automan/automation.py

```diff
@@ -1174,6 +1177,12 @@ def _setup_argparse(self):
             dest='update_remote', default=False,
             help='Update remote worker machines.'
         )
+        parser.add_argument(
+            '--rm-remote-output', nargs='*', action="store",
+            dest="rm_remote_output", type=str, default='',
+            help="remove output folder from the mentioned machines"
+            "use 'all' to remove from all host"
```

Comment:
Will this also wipe out any local output?  When you say "all" is localhost included?  Please mention this in the help documentation here.

Reply:
done